packages/react/src/components/Tag/Tag.js

```diff
@@ -48,8 +49,12 @@ const Tag = ({
           ? `${title} ${children}`
           : `Clear filter ${children}`
       }
+      disabled={disabled}
+      onClick={!disabled ? onClick : undefined}
```

Comment:
Does a `button` fire an `onClick` event if it's `disabled`?

Reply:
it doesn't, I think that was left over from a merge conflict resolution